packages/container/lib/container.js

```diff
@@ -379,31 +379,27 @@ function areInjectionsDynamic(injections) {
   return !!injections._dynamic;
 }
 
-function buildInjections() /* container, ...injections */{
+function buildInjections(container, ...injectionsArray) {
   let hash = {};
+  let injections = Array.prototype.concat(...injectionsArray);
```

Comment:
seems like it is, so I don think there is a point to this concat step.

https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2016%2Clatest%2Creact%2Cstage-2&targets=&browsers=&builtIns=false&code=function%20buildInjections(container%2C%20...injectionsArray)%20%7B%0A%20%20return%20injectionsArray%3B%0A%0A%7D%0A&experimental=true&loose=true&spec=false&playground=false